elements/multiple-choice/src/multiple-choice.js

```diff
@@ -436,6 +443,12 @@ class MultipleChoice extends SchemaBehaviors(PolymerElement) {
             title: "Incorrect feedback",
             description: "Feedback when they get it wrong",
             inputMethod: "textfield"
+          },
+          {
```

Comment:
This is HAXschema meaning that it's leverages to build the UI for this element. This is correct schema I'm just pointing out that it's not necessary to operate. HAXschema is worth reading up on because there are parallels to what your doing here (in general).
https://haxtheweb.org/hax-schema for docs
https://github.com/elmsln/lrnwebcomponents/blob/master/elements/hax-body-behaviors/lib/HAXWiring.js for deep dive

Effectively HAXschema ends up being an abstraction of JSON schema so that we still have it conform to a standard yet we're placing constraints on the way other developers leverage that standard in the name of UX uniformity. In your case I can see a similar parallel of the user engagement event system and xAPI. We want an abstraction of xAPI that way we can make meaning out of it more easily without everyone having to mess w/ the raw capabilities (though having those as a fallback is always good)

Reply:
This (Incorrect Feedback) is original to the multiple-choice element.  We'll probably want to discuss what you're thinking here.